docs/guidelines/custom-plugin.md

```diff
@@ -0,0 +1,13 @@
+# Custom plugin
+
+## Overview
+
+Read the document to find out how to write custom plugin for `documentation-component`.
+
+## Project structure
+
+You can use exemplary project/code structure for custom plugin [here](../templates/project-structure-templates/plugin).
+
+## Example
+
+See exemplary plugin written in TypeScript for `markdown` content [here](https://github.com/kyma-incubator/documentation-component/tree/master/packages/markdown-render-engine/src/plugins/tabs).
```

Comment:
Maybe we should make some kind of example plugin(s), because tabs plugin has some kind of nasty logic and code in there, better not show it as an example 😄

Reply:
and watch out for absolute link